src/runtime/security/server_negotiation.cpp

```diff
@@ -17,14 +17,22 @@
 
 #include "server_negotiation.h"
 
+#include <dsn/dist/fmt_logging.h>
+
 namespace dsn {
 namespace security {
 
-server_negotiation::server_negotiation(rpc_session *session) : negotiation(session) {}
+server_negotiation::server_negotiation(rpc_session *session) : negotiation(session)
+{
+    _name = fmt::format("S_NEGO_L({})=>R({})",
```

Comment:
```suggestion
    _name = fmt::format("S_NEGO_L({})=>R({})",
```

this name is difficult to understand.

Reply:
How about `client_negotiation({})=>({})` ?

Reply:
```suggestion
    _name = fmt::format("SERVER_NEGOTIATION(CLIENT={})")",
```
The server address is redundant. Likewise for the name of client_negotiation. I don't actually understand the need of “name”. What name is used for?